core/meta/object-property.js

```diff
@@ -32,28 +31,28 @@ var ObjectProperty = exports.ObjectProperty = Montage.specialize( /** @lends Obj
     },
 
     /**
-     * Add all the properties defined in the blueprint to the target prototype.
+     * Add all the properties defined in the object descriptor to the target prototype.
      *
-     * If the blueprint is null, this method will make a best attempt to locate
+     * If the object descriptor is null, this method will make a best attempt to locate
      * it.
      *
      * @function
      * @param {Object} prototype
      * @param {Blueprint} blueprint
```

Comment:
the param should change from {Blueprint} to {ObjectDescriptor}, there are  few others like that on param/return